oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -171,6 +171,8 @@ volunteers are needed, we will put out a call to join. Volunteers will need to a
 and will then be evaluated by the existing working group members. We hope to have six to seven members of the working
 group at a time.
 
+There will also be ways to participate in the security of the Open edX platform without being a member of the working group. Maintainers and core contributors of the Open edX community interested in volunteering to implement security hotfixes for other maintainers who may not have the bandwidth to immediately address vulnerabilities are encouraged to email their interest to security@openedx.org.
```

Comment:
@alangsto @feanil Wanted to draw your attention to this line that I added post-PR-creation to make sure you saw it and provide your thoughts!